samples/proxy_sample/README.md

```diff
@@ -0,0 +1,449 @@
+#Azure IoT Gateway SDK - Hello World
+
+This document provides a detailed overview of the Hello World sample [code](./src) which uses the fundamental components of the Azure IoT Gateway SDK architecture to log a hello world message to a file every 5 seconds.
```

Comment:
Is this whole file correct? This seems like the README.md from the Hello World Sample.